src/SampleLibraryTests/Setup.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Globalization;
+using System.IO;
+using System.Reflection;
+using NUnit.Framework;
+
+
+[SetUpFixture]
+public class Setup
+{
+    private string moduleRootFolder;
+    List<string> resolutionPaths;
+
+    [OneTimeSetUp]
+    public void RunBeforeAllTests()
+    {
+        var thisDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        var configPath = Path.Combine(thisDir, "TestServices.dll.config");
```

Comment:
So this config file is in the test nuget package that is a dependency of dynamosamples?

Reply:
YEs. And ideally the content of this file (Setup.cs) would live in the SystemTestServices.dll (or somewhere in Dynamo). But I found it too complicated to do it now